service/frontend/workflow_handler.go

```diff
@@ -5182,8 +5184,47 @@ func (wh *WorkflowHandler) GetDeploymentReachability(
 func (wh *WorkflowHandler) UpdateWorkflowExecutionOptions(
 	ctx context.Context,
 	request *workflowservice.UpdateWorkflowExecutionOptionsRequest,
-) (*workflowservice.UpdateWorkflowExecutionOptionsResponse, error) {
-	return nil, status.Errorf(codes.Unimplemented, "method UpdateWorkflowExecutionOptions not implemented")
+) (_ *workflowservice.UpdateWorkflowExecutionOptionsResponse, retError error) {
+	defer log.CapturePanic(wh.logger, &retError)
+	wh.logger.Debug("Received UpdateWorkflowExecutionOptions")
```

Comment:
I don't know if you want this line. Probably for pre-release it can be useful. Add `tag.WorkflowID()` to it, please.

Reply:
I think since we are not expecting this to be called in the "happy path" of using versioning, I'm interested to see when it is used.. but tbh this is not that informative. I'll keep for pre-release and add the tag